src/index.html

```diff
@@ -3,11 +3,309 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;700;800&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <h1 hidden>Bose</h1>
+    <header class="header" id="header">
+      <div class="header__wrapper">
+        <div class="header__menu">
+          <a href="#home" class="header__logo" >
+            <img
+              src="./images/logo.svg"
+              alt="BOSE logo"
+              class="header__logo-img"
+            >
+          </a>
+          <div class="header__navigation">
+            <a
+              href="tel:+1 234 5555-55-55"
+              class="header__phone"
+              target="_blank"
+            >
+          </a>
+            <button class="header__menu-button header__burger"></button>
```

Comment:
you got an issue - when menu is opened and menu item is clicked - you're navigated to a href'ed section, but scroll remains disabled. Try it.
And fix please )